scss/main.scss

```diff
@@ -31,38 +31,17 @@ p {
 }
 
 .btn-link {
-  cursor: pointer;
-  padding: 0;
+  &.inline-link {
```

Comment:
We should only use `<a>` for hyperlinks. I can rename the class name if it's confusing.